### PR TITLE
Quicksort java

### LINKED ISSUE #150: [Algo] Quick Sort [Java]
https://en.wikipedia.org/wiki/Quicksort

Please assign this to me.

### PR DESCRIPTION
**Pull request by Aleksandren**
quicksort.java Add Quicksort on Java

Closed #150

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

Fixes #{150}

**By submitting this pull request I confirm I've read and complied with the
below declarations.**

- [x] I have read the [Contribution guidelines](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [commit guidelines](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md#write-good-commit-messages) for this project.
- [x] My code follows the [standard code structure](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md#samples-code).
- [x] This pull request has a descriptive title. For example, `{Tag}: Add {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.
- [x] This pull request shall only be reviewed and merged once all the checks passes. No maintainer or supporter shall be obliged to review it before this condition is met.
- [x] I have mentioned the issue number correctly (with hyperlink) in this pull request description.

After you submit your pull request, **DO NOT** click the 'Update Branch' button.

### CODE REVIEW

File: bubble_sort/Python/Bubble_sort.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+def bubble_sort(mas):
+    for i in range(len(mas)):
+        for j in range(len(mas) - 1, i, -1):
+            if mas[j] < mas[j-1]:
+                mas[j], mas[j-1] = mas[j-1], mas[j]
```
Comment (sangamcse) [inline bubble_sort/Python/Bubble_sort.py]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpd1qdvjsf/bubble_sort/Python/Bubble_sort.py
+++ b/tmp/tmpd1qdvjsf/bubble_sort/Python/Bubble_sort.py
@@ -4,4 +4,6 @@
             if mas[j] < mas[j-1]:
                 mas[j], mas[j-1] = mas[j-1], mas[j]
     return mas
+
+
 print(bubble_sort([4, 3, 2, 1]))
```

File: bubble_sort/Python/Bubble_sort.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    return mas
+print(bubble_sort([4, 3, 2, 1]))
```
Comment (sangamcse) [inline bubble_sort/Python/Bubble_sort.py]: E305 expected 2 blank lines after class or function definition, found 0

*Origin: PycodestyleBear (E305), Section: `all.autopep8`.*

**Comment by sangamcse** [comment]
Comment on b768198d47c54a43fb278ea3ff2a638f0ab0049d, file bubble_sort/Python/Bubble_sort.py, line 6.

The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp7rkcukyt/bubble_sort/Python/Bubble_sort.py
+++ b/tmp/tmp7rkcukyt/bubble_sort/Python/Bubble_sort.py
@@ -4,4 +4,6 @@
             if mas[j] < mas[j-1]:
                 mas[j], mas[j-1] = mas[j-1], mas[j]
     return mas
+
+
 print(bubble_sort([4, 3, 2, 1]))
```

**Comment by sangamcse** [comment]
Comment on b768198d47c54a43fb278ea3ff2a638f0ab0049d, file bubble_sort/Python/Bubble_sort.py, line 7.

E305 expected 2 blank lines after class or function definition, found 0

*Origin: PycodestyleBear (E305), Section: `all.autopep8`.*

**Comment by sangamcse** [comment]
@Aleksandren, It shows that you haven't read the contribution guidelines

**Comment by Aleksandren** [comment]
@sangamcse, I will issue in accordance with the requirements

**Comment by sangamcse** [comment]
Closing since @udayvig asked for it first.

**Pull request closed without merging**